addon/components/polaris-drop-zone.js

```diff
@@ -280,73 +283,67 @@ export default Component.extend(ContextBoundEventListenersMixin, {
    * @public
    * @property onFileDialogClose
    */
-  onFileDialogClose() {},
-
-  iconDragDrop,
-
-  iconAlertCircle,
-
-  state: computed(() => State.create()).readOnly(),
+  onFileDialogClose() {}
 
-  isDragging: or('active', 'state.dragging').readOnly(),
+  iconDragDrop = iconDragDrop;
+  iconAlertCircle = iconAlertCircle;
+  dragTargets = [];
 
-  dropZoneClasses: computed(
-    'outline',
-    'isDragging',
-    'state.error',
-    'sizeClass',
-    function() {
-      let { outline, isDragging, state, sizeClass } = this.getProperties(
-        'outline',
-        'isDragging',
-        'state',
-        'sizeClass'
-      );
+  @computed()
```

Comment:
do we want to keep the `.readOnly()` on all these CPs?

Reply:
no, having a computed property with only a getter....trying to set it, should throw an error